packages/console/src/components/ErrorBoundary/index.module.scss

```diff
@@ -0,0 +1,34 @@
+@use '@/scss/underscore' as _;
+
+.container {
+  overflow-y: auto;
+  height: 100vh;
+
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--color-layer-1);
+    color: var(--color-text);
+    padding: _.unit(6);
+    width: 858px;
+    min-height: 100%;
+    margin: 0 auto;
```

Comment:
`align-items: center` ?

Reply:
No, the exception call stack should be left aligned, only the top illustration image and the H2 title need to be center aligned